pom.xml

```diff
@@ -59,29 +48,483 @@
   </distributionManagement>
 
   <properties>
-    <scm.gitBaseUrl>https://github.com/openhab</scm.gitBaseUrl>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <oh.java.version>1.8</oh.java.version>
+    <maven.compiler.source>${oh.java.version}</maven.compiler.source>
+    <maven.compiler.target>${oh.java.version}</maven.compiler.target>
+    <maven.compiler.compilerVersion>${oh.java.version}</maven.compiler.compilerVersion>
+
+    <bnd.version>4.1.0</bnd.version>
+    <karaf.version>4.2.1</karaf.version>
+    <sat.version>0.5.0</sat.version>
+    <slf4j.version>1.7.21</slf4j.version>
+    <xtext.version>2.14.0</xtext.version>
   </properties>
 
+  <repositories>
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>jcenter</id>
+      <name>JCenter Repository</name>
+      <url>https://jcenter.bintray.com</url>
+    </repository>
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>esh-3rdparty-mvn</id>
+      <name>openHAB Core :: 3rd party Maven Repository</name>
+      <url>http://eclipse.github.io/smarthome/third-party/m2-repo/</url>
+    </repository>
+    <repository>
+      <id>bintray-maggu2810-maven</id>
+      <url>https://dl.bintray.com/maggu2810/maven/</url>
+    </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <id>jcenter</id>
+      <url>https://jcenter.bintray.com</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.openhab.core.bom</groupId>
+        <artifactId>org.openhab.core.bom.compile</artifactId>
+        <version>${project.version}</version>
+        <type>pom</type>
+        <scope>provided</scope>
```

Comment:
I needed to change the scope to `compile` to be able to build it with maven on the command line. Same for provided below on `compile-model`

Reply:
Really?
I build the whole repository multiple times now and it works.

If the build does not work for you (using the current master), it would perhaps make sense to create an issue with the whole log.

`provided` should be present on compile time (similar to `compile`) but can be missing at `runtime` (different to `compile`).